test/util/constants.js

```diff
@@ -1,8 +1,8 @@
 // Some periods in milliseconds
```

Comment:
Should say seconds instead of milliseconds

Reply:
yes, pushing now